lighthouse-core/config/config.js

```diff
@@ -363,6 +366,45 @@ class Config {
     const configJson = this; // eslint-disable-line no-unused-vars
   }
 
+  /**
+   * Provides a cleaned-up, stringified version of this config. Gatherer and
+   * Audit `implementation` and `instance` do not survive this process.
+   * @return {string}
+   */
+  print() {
```

Comment:
🚲 🏠 : `print` makes me think it will print it not just return the string :) 

`formatAsString` `getPrintString` `stringify` `getDisplayString`

any of those strike your fancy?

Reply:
going with `getPrintString` unless anyone has other votes :) I don't care a whole lot, though it would be nice if whatever we choose communicates that the string result was functional in nature, not just pretty printing or something